.gitlab/ci/on-push.yml

```diff
@@ -7,6 +7,22 @@
   - echo "======= Download configuration ========"
   - ./Tests/scripts/download_demisto_conf.sh  >> $ARTIFACTS_FOLDER/logs/installations.log
 
+trigger-private-build:
+  needs: []
+  stage: unittests-and-validations
+  extends:
+    - .default-job-settings
+  rules:
+    - if: '$FORCE_BUCKET_UPLOAD != "true" && $BUCKET_UPLOAD != "true" && $SKIP_PRIVATE_BUILD != "true"'
```

Comment:
In which workflows should this job run?

Reply:
in general every time lint runs unless force_bucket_upload, bucket_upload, skip_private_build
I changed it now so it would be more readable